_episodes/01-introduction.md

```diff
@@ -64,13 +65,14 @@ Unsurprisingly, software installation and configuration challenges can have
 negative consequences for research: 
 - you can't use a specific tool at all, because it's not available or installable. 
 - you can't reproduce your results because you're not sure what tools you're actually using. 
-- you can't access extra resources because you're not able to replicate your software set up. 
+- you can't access extra/newer resources because you're not able to replicate your software set up. 
+- others cannot validate and/or build upon your work because they cannot recreate your system's unicorn configuration.
```

Comment:
Hell, **I'm** not familiar with the Unicorn software! 😆   

I meant "unicorn" as a synonym for "unique/rare/one-of-a-kind" that would emphasize it's singular existence in all possible universes across time and space.  I'll surround it with quotes to prevent confusion.

Reply:
Ok, maybe it's not that well known! 😆 I realised what you meant, happy to leave this in its original state.